jsonnet/rules.libsonnet

```
@@ -331,6 +331,10 @@ function(params) {
           // Returns 1 if all control plane nodes are ready 0 otherwise.
           // Should be used to suppress alerts during control plane upgrades or disruption.
         },
+        {
+          expr: 'max by (profile) (cluster_monitoring_operator_collection_profile)',
```

We only need to report the selected profile.
```suggestion
          expr: 'max by (profile) (cluster_monitoring_operator_collection_profile == 1)',
```